standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/columnstats/aggr/BinaryColumnStatsAggregator.java

```diff
@@ -60,4 +60,8 @@ public ColumnStatisticsObj aggregate(List<ColStatsObjWithSourceInfo> colStatsWit
     statsObj.setStatsData(columnStatisticsData);
     return statsObj;
   }
+
+  @Override protected ColumnStatisticsData initColumnStatisticsData() {
+    throw new UnsupportedOperationException("initColumnStatisticsData not supported for binary statistics");
```

Comment:
nit: the binary type supports ColumnStatisticsData, the exception and the method name makes me confusing, can we change the method name or add more details why it doesn't support?

Reply:
You are right, the method does not do much for `binary` and `boolean`, but it still make sense, so I have:
- removed the exception, replaced with `return new ColumnStatisticsData();`
- used the method to actually initialize the emtpy `ColumnStatisticsData` for those two data types